documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc

```diff
@@ -98,6 +102,11 @@ The default is `365` days.
 The default is `30` days to initiate certificate renewal before the old certificates expire.
 <14> (Optional) The Java options used by the JVM running the User Operator
 <15> (Optional) The debugging (`-D`) options set for the User Operator
+<16> (Optional) Prefix which will be used in the names of Kubernetes secrets created by the User Operator
+<17> (Optional) Indicates whether the Kafka cluster supports management of authorization ACL rules using the Kafka Admin API.
+When set to `false`, the User Operator will reject all resources with `simple` authorization ACL rules.
+This helps to avoid unnecessary exceptions in the Kafka cluster logs.
+The default is `true`.
```

Comment:
Maybe some context about when to set and not set this in https://strimzi.io/docs/operators/in-development/using.html#con-securing-client-authorization-str

> **ACL management**
> Implementation of ACL management depends on your client authorization configuration. 
> 
> - For simple authorization, it's always enabled. 
> - For OPA authorization, it's always disabled. 
> - For Keycloak authorization, you can delegate to the simple authorizer in the configuration. This means you can use Keycloak authorization and ACLs. 
>  
> If ACL managment is not enabled, Strimzi throws an error if a resource contains ACLs.
> 
> If you're using a standalone deployment of the User Operator, use configuration to manage ACLs. Enabled by default, you can disable ACL management using the `STRIMZI_ACLS_ADMIN_API_SUPPORTED` environment variable.

Reply:
@PaulRMellor I updated it, but didn't use exactly the same text as you suggested above. Could you please have another look?

Reply:
@scholzj  Thanks Jakub. The custom option completes it. Looks good. Just a few minor suggestions.